docs/Database_Schema.md

```diff
@@ -152,6 +154,8 @@ CREATE TABLE IF NOT EXISTS "tx_outputs" (
     "output_index"          INTEGER NOT NULL,
     "tx_hash"               BLOB    NOT NULL,
     "utxo_key"              BLOB    NOT NULL,
+    "type"                  INTEGER NOT NULL,
```

Comment:
If this is a transaction type, you can refer to the transaction table.

Reply:
The conditions below were added when recording TxOutput.
                let unlock_height: JSBI;
                let tx_type: TxType;
                if (melting && address != TxPayloadFee.CommonsBudgetAddress)
                {
                    tx_type = tx.type;
                    unlock_height = JSBI.add(height.value, JSBI.BigInt(2016));
                }
                else if (tx.type == TxType.Freeze && tx.outputs.length >= 2 &&
                    (out_idx == 0 && JSBI.lessThan(output.value, JSBI.BigInt(400_000_000_000))))
                {
                    tx_type = TxType.Payment;
                    unlock_height = JSBI.add(height.value, JSBI.BigInt(1));
                }
                else
                {
                    tx_type = tx.type;
                    unlock_height = JSBI.add(height.value, JSBI.BigInt(1));
                }

Reply:
The type of table tx_outputs can have a different value than the type of table transactions.
The type of transaction in the transactions type.
This function was added from Agora to prevent the refund output of the transaction from being frozen.
Agora has a type in UTXO, and I added it to the table.

Reply:
Oh, I think we will make a separate UTXOs table.
I think the update query will happen again and again.
Frequent updates can cause deadlocks.
We recommend that you use the Transactions table as read-only.

Reply:
It has been modified by adding a table that only manages UTXO.